migrations/00007_add_authenticator_nickname.sql

```diff
@@ -0,0 +1,5 @@
+BEGIN;
+ALTER TABLE auth.user_authenticators
+  ADD COLUMN nickname text;
```

Comment:
I think it's better not to have `IF NOT EXISTS` as it could hide underlying issues with the schema.

The `nickname` column should not exist. If it does, this migration will fail, and the underlaying problem must be fixed.

If a `nickname` column with type `int` existed already, this migration would be successful, but then errors would show elsewhere. 

Better to catch these errors early.

Reply:
Because we don't live in a perfect world :) Imagine being able to run all migrations for a broken project and end up at the desired state, versus having to, as an example, get into RDS, and run SQL commands to fix what's broken because users messed with the schema. We do not control what our users do to their databases. But we will have to fix those issues once they ask us to. It is very different having to manage our own database versus having to manage thousands.